mps_youtube/main.py

```diff
@@ -3746,6 +3769,7 @@ def main():
         quits: r'(?:q|quit|exit)$',
         plist: r'pl\s+%s' % pl,
         yt_url: r'url\s(.*[-_a-zA-Z0-9]{11}.*$)',
+        yt_url_file: r'url_file\s([^.]+\.txt$)',
```

Comment:
I'd expect this feature to work with any file extension, not just `.txt`. Maybe something like 
```py
r'url_file\s+([^\s]+$)'
```